api/api.go

```diff
@@ -61,15 +61,24 @@ func Consulize(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	js, err := json.Marshal(out)
+	_, err = json.Marshal(out)
```

Comment:
I think it would look better like this:
 
```
if _, err = json.Marshal(out); err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
}
```
What do you think? :slightly_smiling_face:

Reply:
Your approach is nicer! :-)

But, eventually we are going to use the output of the `json.Marshal` (currently working on that ;-))

Reply:
That makes sense!